md/bdm-access-control.md

```diff
@@ -314,3 +314,153 @@ Deploy your access control file.
+For the details on how to easily create and map those profiles see the [Profile Creation](profileCreation.md) page.
+
+**3. Create a process and generate some request instances**
+
+You will need some instances of the request object, as well as some students. To create them, follow the steps discribed in the section  **II - Invoice**, part 3 *Create a process and generate some data* of this very page.
+
+**4. Create the visualisation application**
+
+We will now create an application for the teachers to review the student requests : 
+
+In the UI Designer, create an application page (*reviewRequests*):
+
+- Create 2 new variables
+    - **Name** : requestList
+    - **Type** : External API
+    - **API URL**: ../API/bdm/businessData/com.company.model.Request?q=findBySubject&p=0&c=10&f=subject%3D{{selectedSubject}}
+    
+    -  **Name** : selectedSubject
+	-  **Type** : String
+    
+- Add a new Select box to the page (To choose beetween subjects):
+    - **Label** : Subject class
+    - **Available Values** : Mathematics, Physics (constants).
+    - **Value** : selectedSubject
+    
+- Add  a new Table widget to the page (To display the requests):
+    - **Headers** : Id, Subject, Content, Medical comment, Student (constants)
+    - **Content** : requestList (script)
+    - **Column keys** : persistenceId, subject, content, medicalComment, student.fullname
+    
+Now in the Studio, create a new [application](applicationCreation.md). Call it *TeacherApp*, give it a theme, and a homepage token, and map it to the *Teacher* profile.
+Then create a one-page menu *Student Request*, with your homepage token and add to it the page you just created in the UID.
+
+If you preview the page at this point, both jan.fischer and helen.kelly can access your new application, select from a drop down list either *Mathematics* or *Physics*, wich should display all the Mathematics and all the Physics requests, respectively, for both users.
+
+**5. Security implementation**
+
+- Got to *BonitaStudioSubscription-7.7.0/workspace/tomcat/setup/*
+- Run *setup pull*
+- Go the */BonitaStudioSubscription-7.7.0/workspace/tomcat/setup/platform_conf/current/tenants/1/tenant_portal/* folder that should have just appeared.
+- Open the *dynamic-permissions-checks-custom.properties*, and add the following line:
+``` GET|bdm/businessData/com.company.model.Request=[check|SubjectTeacherPermissionRule] ```
+- This line indicates that before executing any queries on the com.company.model.Request object types in the BDM, a verification has to be run. In this case we indicate that the verification is a groovy script, *SubjectTeacherPermissionRule.groovy*, which we will now create.
+- Go to */BonitaStudioSubscription-7.7.0/workspace/tomcat/setup/platform_conf/current/tenants/1/tenant_security_scripts*. Create a file *SubjectTeacherPermissionRule.groovy*, with the following content:
+
+```groovy
+package org.bonitasoft.permissions
```

Comment:
remove the package declaration. we did it on your IntelliJ installation, but in real life, there must be NO package (default unnamed package)